bin/ErrorHelpers.js

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
Seem to be weird to have `errorHelpers` and `ErrorHelpers`.

Reply:
it should be `errorHelpers` right?